robottelo/config/settings.py

```diff
@@ -0,0 +1,644 @@
+"""Define and instantiate the configuration class for Robottelo."""
+import logging
+import os
+import sys
+import urlparse
+
+from nailgun import entities, entity_mixins
+from nailgun.config import ServerConfig
+
+from robottelo import test_data
+
+try:
```

Comment:
Can you add a comment explaining that this is needed to handle python 2.7 and 3.0+